core/api_backend.go

```diff
@@ -82,3 +85,23 @@ func (b *HmyAPIBackend) HeaderByNumber(ctx context.Context, blockNr rpc.BlockNum
 func (b *HmyAPIBackend) GetPoolNonce(ctx context.Context, addr common.Address) (uint64, error) {
 	return b.txPool.State().GetNonce(addr), nil
 }
+
+// SendTx ...
+func (b *HmyAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction) error {
+	return b.txPool.Add(ctx, signedTx)
```

Comment:
txPool is not used in our blockchain - by not blocking  it you an add TODO to fix this logic later.